### PR TITLE
Add support for Kratos 0.6

### PR DESCRIPTION
**Pull request by anonrig**
Fixes https://github.com/ory/kratos/issues/1351

### CODE REVIEW

**Comment by anonrig** [comment]
Done and done @aeneasr

File: helm/charts/kratos/templates/configmap-templates.yaml [hunk 1]
```diff
@@ -17,6 +17,10 @@ data:
   {{- if $resultEntry.body }}
   "email.body.gotmpl": |-
 {{ $resultEntry.body | indent 4 }}
+  {{- end }}
+    {{- if $resultEntry.plainBody }}
+  "email.body.plaintext.gotmpl": |-
+{{ $resultEntry.plainBody | indent 4 }}
```
Comment (aeneasr) [inline helm/charts/kratos/templates/configmap-templates.yaml:23]:
```suggestion
    {{- end }}
    {{- if $resultEntry.plainBody }}
  "email.body.plaintext.gotmpl": |-
{{ $resultEntry.plainBody | indent 4 }}
```

File: helm/charts/kratos/templates/configmap-templates.yaml [hunk 2]
```diff
@@ -17,6 +17,10 @@ data:
   {{- if $resultEntry.body }}
   "email.body.gotmpl": |-
 {{ $resultEntry.body | indent 4 }}
+  {{- end }}
+    {{- if $resultEntry.plainBody }}
```
Comment (aeneasr) [inline helm/charts/kratos/templates/configmap-templates.yaml]:
```suggestion
  {{- end }}
  {{- if $resultEntry.plainBody }}
```

File: helm/charts/kratos/templates/configmap-templates.yaml [hunk 3]
```diff
@@ -17,6 +17,10 @@ data:
   {{- if $resultEntry.body }}
   "email.body.gotmpl": |-
 {{ $resultEntry.body | indent 4 }}
+    {{- end }}
+    {{- if $resultEntry.plainBody }}
```
Comment (aeneasr) [inline helm/charts/kratos/templates/configmap-templates.yaml:21]:
```suggestion
  {{- end }}
  {{- if $resultEntry.plainBody }}
```

**Pull request merged**